### PR TITLE
Add nukeeper website structure

### PR DESCRIPTION
**Pull request by MarcBruins**
This adds the nukeeper website structure.  It's a [Hugo website](https://gohugo.io/) which uses the [Learn theme](https://learn.netlify.com/en/).

It enables us to update the website as markdown files and have docs update with every breaking change/feature update that we have. The markdown is hosted beside the code in a `site` folder, every change to the .MD file regenerates the website.

I suggest we take this as a base and iterate upon it, at the time we are comfortable with the website and all Wiki content is moved we can close the wiki.

### CODE REVIEW

File: site/config.toml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+languageCode = "en-us"
+title = "NuKeeper"
+theme = "hugo-theme-learn"
+
```
Comment (slang25) [inline site/config.toml]:
```suggestion
enableEmoji = true
```

Comment (slang25) [reply site/config.toml]: This will show the emoji checkmarks where we use them

Comment (MarcBruins) [reply site/config.toml]: wow we definitely need that!

Comment (MarcBruins) [reply site/config.toml]: done 👍 ⚡️

**Pull request merged**